src/Hl7.Fhir.Specification/Specification/Snapshot/ElementDefnMerger.cs

```diff
@@ -172,7 +173,7 @@ void merge(ElementDefinition snap, ElementDefinition diff, bool mergeElementId,
                 snap.Binding = mergeBinding(snap.Binding, diff.Binding);
 
                 // [MV 20220803] Remove Binding when the element has no bindable type
-                if (snap.Binding is not null && !snap.Type.Any(t => ModelInfoNEW.IsBindable(t.Code)))
+                if (snap.Binding is not null && !snap.Type.Any(t => ModelInfoExtensions.IsBindable(t.Code)))
```

Comment:
Idem, we need to assume all bindeable types are in common. Which is probably true, or at least as true as in the old situation. (I can make up exceptions - but the "old" SDK did not support that either)

Reply:
Indeed all bindeable types are in Common